tests/unit_tests/memory/chat_message_histories/zep_test.py

```diff
@@ -0,0 +1,66 @@
+# type: ignore
+import pytest
+from pytest_mock import MockerFixture
+from zep_python import Memory, Message, Summary, ZepClient
+
+from langchain.memory.chat_message_histories import ZepChatMessageHistory
+from langchain.schema import AIMessage, HumanMessage
+
+
+@pytest.fixture
+def zep_chat(mocker: MockerFixture) -> ZepChatMessageHistory:
+    mock_zep_client: ZepClient = mocker.patch("zep_python.ZepClient", autospec=True)
+    zep_chat: ZepChatMessageHistory = ZepChatMessageHistory(
+        "test_session", "http://localhost:8000"
+    )
+    zep_chat.zep_client = mock_zep_client
+    return zep_chat
+
+
+def test_messages(mocker: MockerFixture, zep_chat: ZepChatMessageHistory) -> None:
```

Comment:
any unit tests that require optional dependencies (all the ones here) need a decorator
```python
@pytest.mark.requires("zep_python")
```

Reply:
Thanks. Zep tests are now passing in `core`, but the extended tests are failing as it looks like zep is still a dependency despite having removed zep from the `extended_tests` section of the pyproject file.

```
tests/unit_tests/memory/chat_message_histories/zep_test.py ssssss        [ 55%]
```

Reply:
ah you'll actually want to keep it in `extended_tests` and remove it from `all`. and then run `poetry lock --no-update`

also run `make format` to fix linting issues

and could we rename zep_test.py -> test_zep.py (to match naming convention for other test files)

Reply:
Thanks. Just done the above. Let's see if the tests pass :-)

Reply:
That did the trick! Thanks @dev2049